lms/djangoapps/course_api/blocks/transformers/blocks_api.py

```diff
@@ -65,3 +67,5 @@ def transform(self, usage_info, block_structure):
         BlockCountsTransformer(self.block_types_to_count).transform(usage_info, block_structure)
         BlockDepthTransformer(self.depth).transform(usage_info, block_structure)
         BlockNavigationTransformer(self.nav_depth).transform(usage_info, block_structure)
+        if ENABLE_VIDEO_URL_REWRITE.is_enabled(block_structure.root_block_usage_key.course_key):
+            VideoBlockURLTransformer().transform(usage_info, block_structure)
```

Comment:
Please update the comment above from

```
Note: BlockDepthTransformer must be executed before BlockNavigationTransformer.
```

to say: 
```
Note:
* BlockDepthTransformer must be executed before BlockNavigationTransformer.
* StudentViewTransformer must be executed before VideoBlockURLTransformer.
```
This allows us to track interdependencies.